topics/metagenomics/tutorials/foodborne-pathogen-detection-nanopore-sequencing-data/tutorial.md

```diff
@@ -61,11 +61,15 @@ In this tutorial, the datasets are all generated by Biolytix. They are all chick
 
 > <hands-on-title>Data upload</hands-on-title>
 >
-> 1. Create two new histories for this tutorial one for every sampling time; name them `Barcode10`and `Barcode11`
```

Comment:
It seems that you are using the same file for the short and the long version? Perhaps I'm wrong sorry.

Reply:
Yeah both files are the same the only difference between the long and short version is that the tool by tool is explained in the long version where you can run a tool by tool, however the shortversion we run the workflows instead. But we are doing the same things with the same files and same results

Reply:
I actually wanted to thank you for all the comments <3 and the explanation of the AMR and VFs